src/eq_schema/RoutingConditions.js

```diff
@@ -8,33 +10,62 @@ const getAllOptions = condition => {
   }
 };
 
-const createOptionByIdLookup = flow(
+const keyOptionsById = flow(
   flatMap(getAllOptions),
   keyBy("id")
 );
 
+const RoutingConditionBuilderError = {
+  becauseAnswerTypeIsNil: () => {
+    throw new Error("Answer type is null or undefined");
+  },
+  becauseAnswerTypeIsUnsupported: answerType => {
+    throw new Error(
```

Comment:
I would prefer if the throw was made by the caller so it was obvious to the reader of the calling function what was going on